### DISCUSSION TITLE
Citizen 智能手錶結合 NASA 和 IBM AI 來監測你的警覺度

### DISCUSSION THREAD

**Discussion by awehao** [1101-GS3073-C/Python-Community-GS3073-C#570] [General]
竟然連老牌製錶都開始結合科技!我覺得，除了現有的小米及applewatch許多錶類公司也跟進推出智能手錶，結合強大工具，讓手錶除了以往時間的功能外，更能改善生活品質，雖然提高警覺性看似一個新飲的功能，但具體是怎樣還得等發布之後才能下定論
https://chinese.engadget.com/citizens-new-cz-smart-watch-uses-ai-to-determine-your-level-of-alertness-133016557.html

**Comment by number32984** [comment]
applewatch真是個好用又有質感的東西，雖然有點小貴

**Comment by Chenszuhong** [comment]
哇 這超帥的 我本來就蠻喜歡星城這個手錶品牌 在結合那些感覺很厲害的功能感覺就很帥

**Comment by magicmicheal** [comment]
我記得一些嫆來潛水以及登山的錶都十分需要結合AI，透過AI監測是否有危及到身命特徵的時刻，還殼已發送求救訊息給外界，潛水時會時刻琛測氧氣筒的含氧量
